packages/project-access/src/types/info/index.ts

```diff
@@ -91,3 +91,12 @@ export type AppType =
     | 'SAPUI5 Extension'
     | 'Fiori Reuse'
     | 'Fiori Adaptation';
+
+export interface JSONFileInfo {
+    // Spaces size
+    size?: number;
+    // Use '\t' symbol instead of space
+    useTabSymbol?: boolean;
+    // End of file
```

Comment:
Could we simplify this to one property `space: number | string` which is the same as described in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#space?

```suggestion
    // Space as defined in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#space
    sapce: number | string;
```